src/pages/junsobKim/Main/COMPANYDETAIL.js

```diff
@@ -0,0 +1,17 @@
+const COMPANYDETAIL = [
```

Comment:
```suggestion
const COMPANY_DETAIL = [
```
상수데이터의 네이밍 컨벤션은 UPPER_CASE입니다